libs/application-generic/src/usecases/tier-restrictions-validate/tier-restrictions-validate.usecase.ts

```diff
@@ -213,6 +207,27 @@ function buildIssue(
   return null;
 }
 
+const getFlagNameByType = (stepType: StepTypeEnum.DELAY | StepTypeEnum.DIGEST) => {
```

Comment:
Nit-picky: This function makes the code more cumbersome that it should be. I suggest calling the feature flag service and the tier service with the necessary arguments per step in the function. Keeping it dry doesn't help too much.

Reply:
actually, i made it because i was sure i would reuse it, but it used only one!